src/OSPSuite.UI/Views/Charts/ChartEditorView.cs

```diff
@@ -80,7 +80,7 @@ public ChartEditorView(IMenuBarItemToBarItemMapper barItemMapper, IImageListRetr
          repositoryItemCheckEditForUsedIn.ValueGrayed = null;
 
          _barEditItemLinkSimulationObserved.EditValue = false;
-         //_barEditItemLinkSimulationObserved.Visibility = BarItemVisibility.Never;
+         _barEditItemLinkSimulationObserved.Visibility = BarItemVisibility.Never;
```

Comment:
use SetlinkSimDataMenuItemVisisbility

Reply:
also rename SetlinkSimDataMenuItemVisisbility to remove the type
Are you using resharper? it should show you typo like this out of the box